### PR TITLE
ZENKO-1051 bf: Add sentinel support to backbeat entrypoint

### PR DESCRIPTION
**Pull request by tmacro**

### CODE REVIEW

**Review by dora-korpar — APPROVED** [review]

File: conf/Config.js [hunk 1]
```diff
@@ -79,6 +79,16 @@ class Config extends EventEmitter {
         healthChecks.allowFrom =
             healthChecks.allowFrom.concat(defaultHealthChecks);
 
+        if (parsedConfig.redis &&
+          typeof parsedConfig.redis.sentinels === 'string') {
+            let redisConf = { sentinels: [], name: parsedConfig.redis.name };
+            parsedConfig.redis.sentinels.split(',').forEach(item => {
+                const [host, port] = item.split(':');
```
Comment (philipyoo) [inline conf/Config.js]: only potential problem is we don't validate that the string will be correctly formed

Should we inject a joi validation in here?

File: conf/Config.js [hunk 2]
```diff
@@ -79,6 +79,16 @@ class Config extends EventEmitter {
...
+                redisConf.sentinels.push({ host, port });
```
Comment (philipyoo) [inline conf/Config.js]: port here will be a string, maybe need to convert to number

File: conf/Config.js [hunk 3]
```diff
@@ -79,6 +79,16 @@ class Config extends EventEmitter {
         healthChecks.allowFrom =
             healthChecks.allowFrom.concat(defaultHealthChecks);
 
+        if (parsedConfig.redis &&
+          typeof parsedConfig.redis.sentinels === 'string') {
+            const redisConf = { sentinels: [], name: parsedConfig.redis.name };
+            parsedConfig.redis.sentinels.split(',').forEach(item => {
+                const [host, port] = item.split(':');
+                redisConf.sentinels.push({ host, port: parseInt(port, 10) });
```
Comment (rahulreddy) [inline conf/Config.js]: Please use `Number.pasrseInt` as the JS global is kinda looked down upon due it's magic behavior.

**Review by philipyoo — APPROVED** [review]

**Comment by rahulreddy** [comment]
I restored the removal of localcache config as it's not relevant to this PR.

**Comment by rahulreddy** [comment]
@bert-e reset

**Review by rahulreddy — APPROVED** [review]

**Pull request merged**